src/Evolution/DgSubcell/Tags/TciStatus.hpp

```diff
@@ -0,0 +1,19 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include "DataStructures/DataBox/Tag.hpp"
+#include "DataStructures/Tensor/TypeAliases.hpp"
+
+/// \cond
+class DataVector;
+/// \endcond
+
+namespace evolution::dg::subcell::Tags {
+/// Stores the status of the troubled cell indicator in the element as a
+/// `Scalar<DataVector>` so it can be observed.
+struct TciStatus : db::SimpleTag {
```

Comment:
is this going to be a value per grid point or one value for the entire grid?

Reply:
Technically it's one value per element, but I don't know how to visualize that in ParaView, so I've opted for making it a `Scalar<DataVector>` so it's easy to visualize. I expect that once things are tested and working we can avoid adding it to the DataBox by default. I think it's also possible to deduce this info just from the mesh, but again, the path of least resistance (by far) is just writing out a `Scalar<DataVector>`. Does that make sense?

Reply:
Yes that makes sense.  I guess my next questions are: 
- Are you printing a size 1 DataMesh or a constant on the active mesh of the element?
- What determines the value printed? (i.e. is it a discrete 0 or 1 depending upon whether or not the Element is troubled, or some continuous value that conveys more information)

Reply:
1. it is the size of the active grid. `N+1` on DG and `2N+1` on subcell
2. it is discrete 1 or 0. I don't really know how to make additional information available easily since each executable has its own set of TCIs

Reply:
I think this is a reasonable approach since it just works with the current infrastructure. @nilsdeppe Do you think it's feasible to add support for single-valued attributes per element to the observers at some point in the future? I think we'll have more applications for that, e.g. for AMR error estimators, or to visualize wall-time distribution and node distribution.

Reply:
From the spectre writing perspective it's not a problem, I don't know what ParaView supports, that could be a bit trickier.

Reply:
Ok, I think if writing such things through the observers works then we'll also be able to pass it along to ParaView. It can do things like color each block in a different color by default, so it should be able to handle this. I looked through https://www.xdmf.org/index.php/XDMF_Model_and_Format but didn't immediately find a solution. I'll open an issue.

Reply:
:+1: